client/my-sites/plan-features/footer.jsx

```diff
@@ -11,7 +11,7 @@ import React, { PropTypes } from 'react';
 import Button from 'components/button';
 import Gridicon from 'components/gridicon';
 
-const PlanFeaturesFooter = ( { translate, current = false, available = true, description, onUpgradeClick = noop } ) => {
+const PlanFeaturesFooter = ( { translate, current = false, available = true, onUpgradeClick = noop } ) => {
```

Comment:
Don't forget to rename this to something like `PlanFeaturesButtons` or `PlanFeaturesActions` as it's not a footer. :)